tools/template-tool/Azure.Sdk.Tools.Template/Azure.Sdk.Tools.Template.csproj

```
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <Description>Template package for testing the tools repelase process</Description>
+    <TargetFramework>net5.0</TargetFramework>
```

net6 or will this cause problems since it's not on the agents yet?
```suggestion
    <TargetFramework>net6.0</TargetFramework>
```

---

Yes it is not on the agent.